sdk/loadtestservice/Azure.Developer.LoadTesting/src/FileUploadOperation.cs

```diff
@@ -0,0 +1,133 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable disable
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Text;
+using System.Text.Json;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Core;
+
+namespace Azure.Developer.LoadTesting
+{
+    /// <summary>
+    /// FileUploadOperation.
+    /// </summary>
+    public class FileUploadOperation : Operation<BinaryData>
+    {
+        private bool _completed;
+        private Response _response;
+        private BinaryData _value;
+        private readonly string _testId;
+        private readonly string _fileName;
+        private readonly LoadTestAdministrationClient _client;
+        private readonly List<string> _terminalStatus = new()
+            {
+                "VALIDATION_SUCCESS",
+                "VALIDATION_FAILURE",
+                "VALIDATION_NOT_REQUIRED"
+            };
+
+        /// <summary>
+        /// Value.
+        /// </summary>
+        public override BinaryData Value => _value;
```

Comment:
This should throw if `HasValue` is false - The [guidelines](https://azure.github.io/azure-sdk/dotnet_implementation.html#dotnet-implement-operation) have more details.